README.asciidoc

```diff
@@ -1051,38 +1083,6 @@ The local web browser can be stopped with `Ctrl-C`.
 
 ================================
 
-==== Custom Console snippets
```

Comment:
I've dropped the docs for this because I think, at least as we have it now, it is super confusing. It isn't compatible with Elasticsearch's tests or `COPY AS cURL` and I expect it is pretty surprising in general.

Reply:
The reason these custom blocks exist is for the Definitive Guide, where we wanted to display a small amount of JSON but then click through to console to show the full example, including setup etc.  Other than the Def Guide, it is not really used anywhere else

Reply:
Yeah! I tracked that down.

We've talked on and off about having some way of hiding something like these custom blocks but in a way with visual feedback for what you are getting. I'm not sure the right way to do it but I could image these making a comeback eventually.

But as I have it now I'm just dropping the docs for them because I don't want people using them without really knowing what they are doing.